src/backend/distributed/sql/citus--9.4-2--9.4-1.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- 9.4-1--9.4-2 was added later as a patch to fix a bug in our PG upgrade functions
+--
+-- This script brings users who installed the patch released back to the 9.4-1
```

Comment:
I think in these downgrade scripts we can mention why we are doing this, something like "We add this unusual path to not skip other intermediate paths while upgrading because we will continue from the mainline".